cmd/kube-scheduler/app/options/options.go

```diff
@@ -198,6 +210,9 @@ func (o *Options) initFlags() {
 
 // ApplyTo applies the scheduler options to the given scheduler app configuration.
 func (o *Options) ApplyTo(logger klog.Logger, c *schedulerappconfig.Config) error {
+	if err := o.ComponentGlobalsRegistry.SetFallback(); err != nil {
```

Comment:
Why call this here, in addition to `Validate`?

Reply:
This is just a fallback in case it is not called in other places. Like in apiserver, `Options` can be used in different clients, there is no guarantee `Validate` will always be called.